### ISSUE TITLE
a11y improvements

### ISSUE THREAD

**Issue by dingo-d** [infinum/wp-boilerplate#44]
We should try to see where the a11y can be improved in the theme. For instance we have

https://github.com/infinum/wp-boilerplate/blob/9f9ed77caae80bdb999c2ea9acd86a021e6d02b4/wp-content/themes/theme_name/index.php#L18

in our `index.php`, which should not be empty - the screen readers won't be able to recognize this.

I'm not 100% sure of the image handling (that should be ok), but in cases where we are using `background-image`we should use some a11y method to notify that that is indeed an image.

**Comment by iruzevic** [comment]
changed code for pagination.

**Comment by dingo-d** [comment]
Fixed in #46
